sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/DotNetSensor.java

```diff
@@ -49,11 +49,13 @@
   private final RoslynDataImporter roslynDataImporter;
   private final DotNetPluginMetadata pluginMetadata;
   private final ReportPathCollector reportPathCollector;
+  private final ProjectTypeCollector projectTypeCollector;
 
-  public DotNetSensor(DotNetPluginMetadata pluginMetadata, ReportPathCollector reportPathCollector,
-    ProtobufDataImporter protobufDataImporter, RoslynDataImporter roslynDataImporter) {
+  public DotNetSensor(DotNetPluginMetadata pluginMetadata, ReportPathCollector reportPathCollector,ProjectTypeCollector projectTypeCollector,
```

Comment:
_Pinkie nitpick:_ Missing space. I have a macro for Ctrl+S to format & save.
```suggestion
  public DotNetSensor(DotNetPluginMetadata pluginMetadata, ReportPathCollector reportPathCollector, ProjectTypeCollector projectTypeCollector,
```